styles/styles.css

```diff
@@ -463,7 +463,6 @@ div ul li {
 
 #getintouch {
   display: grid;
-  width: 100vw;
   height: 100vh;
   background-color: #f5f3f3;
   align-content: center;
```

Comment:
- [ ] Good job on the beautiful design but laying out your element in the form section should be with flex as in the requirements.
  -  In order to lay out the elements on the page you should use Flexbox.